dissect/target/target.py

```diff
@@ -902,8 +902,13 @@ def apply(self) -> None:
                 if getattr(vol, "fs", None) is not None:
                     self.target.filesystems.add(vol.fs)
             except FilesystemError as e:  # noqa: PERF203
-                self.target.log.warning("Can't identify filesystem: %s", vol)
-                self.target.log.debug("", exc_info=e)
+                vol.seek(0)
```

Comment:
If the `seek` or `read` fail, is it ok if we crash here?

Or perhaps add a nested try-catch?

Reply:
If you are sure

 that previously successful seeks and reads guarantee that future seeks and reads succeed,

And there is no value in recovery by logging and skipping the volume,

You can merge this in